spec/components/elements/breadcrumb_nav_component_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Elements::BreadcrumbNavComponent, type: :component do
+  let(:component) { described_class.new(dashboard: true) }
```

Comment:
Since we have `dashboard` declared as a `let` in this scope, I think we could consider:

```suggestion
  let(:component) { described_class.new(dashboard:) }
```

And that would let us make this other change below...